zh/monitor-a-tidb-cluster.md

```diff
@@ -85,11 +85,14 @@ Prometheus 服务默认通过 `NodePort` 暴露，如果 Kubernetes 集群支持
 
 在随 TiDB 集群部署 Prometheus 时，会自动导入一些默认的报警规则，可以通过浏览器访问 Prometheus 的 Alerts 页面查看当前系统中的所有报警规则和状态。
 
-我们目前暂不支持报警规则的自定义配置，如果确实需要修改报警规则，可以手动下载 charts 进行修改。
+我们目前支持报警规则的自定义配置，可以参考下面步骤修改报警规则：
+
+- 参考[通过 TidbMonitor 监控 TiDB 集群](monitor-using-tidbmonitor.md)，在为 TiDB 集群部署监控的过程中，设置 `spec.reloader.service.type` 为 `NodePort` 或者 `LoadBalancer`。
+- 通过 `NodePort` 或者 `LoadBalancer` 访问 reloader 服务，点击上方 `Files` 选择要修改的报警规则文件进行修改，修改完成后 `Save`。
```

Comment:
If these two steps should be taken in order, please use `1, 2,...` to mark the order. 
If the user only has to take either one of the steps, please also specify in the document.